cfn-resources/util/deployment_secret.go

```diff
@@ -64,13 +64,13 @@ func GetAPIKeyFromDeploymentSecret(req *handler.Request, secretName string) (Dep
 		log.Printf("Error --- %v", err.Error())
 		return DeploymentSecret{}, err
 	}
-	fmt.Println(*output.SecretString)
+
 	var key DeploymentSecret
 	err = json.Unmarshal([]byte(*output.SecretString), &key)
 	if err != nil {
 		log.Printf("Error --- %v", err.Error())
 		return key, err
```

Comment:
[q] If there was an error unmarshalling, why are you returning the key here?

Reply:
the key will be empty, i think like this is a bit more clean, than using pointers everywhere